documentation/docs/scenarios/abapEnvironmentAddons.md

```diff
@@ -18,24 +18,32 @@ The installation and maintenance of ABAP software is done / controlled via softw
 !!! caution "Initial Scope"
     The initial scope supports an add-on product consisting of **one** software component. Furthermore, this software component can not be used in multiple add-on products
 
-A software product version is defined by a name and a version string. The name of a software product is a string with a maximum of 30 characters and consists of the namespace and a freely chooseble part - `/NAMESPC/PRODUCT1`. The version string consists of three numbers separated by a dot - `1.2.0`. The numbers in the version string have a hierarchic relationship:
+### Software Product Version
+
+A software product version is defined by a name and a version string. The name of a software product is a string with a maximum of 30 characters and consists of the [namespace](https://launchpad.support.sap.com/#/notes/84282) and a freely chooseble part - `/NAMESPC/PRODUCT1`. The version string consists of three numbers separated by a dot - `1.2.0`. The numbers in the version string have a hierarchic relationship:
 
 - The first number denotes the release. Release deliveries contain the complete scope of functionality. It is possible to change the software component version bundle in a new release.
 - The second number denotes the Support Package Stack level. A Support Package stack consists of Support Package deliveries of the contained software component versions. It is not possible to change the software component version bundle in such a delivery.
-- The third number denotes the Patch level. A Patch delivery contains Patch deliveries of the contained software component versions.
+- The third number denotes the Patch level. A Patch delivery correction deliveries of the included software component versions.
```

Comment:
Did you mean this?
```suggestion
- The third number denotes the Patch level. A Patch delivery corrects deliveries of the included software component versions.
```

Reply:
I think what's bugging me is the " A Patch delivery contains Patch deliveries..", maybe there is another term for patch delivery in case of software product  and software component?